docs/source/user_guide/spyre_overview.md

```diff
@@ -0,0 +1,17 @@
+# IBM Spyre Overview
```

Comment:
Nice!

What do you think about putting this page first and at the top level of the table of contents, like alongside `Getting Started` and `User Guide`? I am not a docs expert, I just have the feeling that we want this info more front and center to anybody looking at these docs

Reply:
Good catch, I considered putting it in its own "Overview" nest above everything, but I think it makes more sense in the "Getting Started" nest, but at the top. Maybe it'll be enough as the first link on the list